search-app/docs/api.md

```diff
@@ -1413,7 +1413,9 @@ Find collections with tag_data in the form of tag_data[tag_key]=tag_value. It fi
 Collections can be found by searching for associated variables. The following variable parameters are supported.
 
 * variable_name
-  * options: pattern
+  * supports `pattern`
```

Comment:
variables are indexed as nested objects in CMR. I think it makes sense to add support for variable_name with the same options that measurement support, but for that, we will need to index variable-names on the collections as the options syntax on inner objects are different. It gets complicated. I think I am going to put this on hold and we can do this after we evaluate our options.